2 - RTB/Documentazione interna/Verbali/23-12-17/23-12-17.typ

```diff
@@ -0,0 +1,42 @@
+
+#import "/template.typ": *
+
+#show: project.with(
+  date: "17/12/23",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+  timeStart: "15:00",
+  timeEnd: "16:00",
+);
+
+= Ordine del giorno
+- analisi Keep doings e Improvements;
+- to do e divisione ruoli.
+
+== Analisi Keep doings e Improvements
+$bold("Keep doings:")$
```

Comment:
Questa formattazione non è conforme con quella usata in precedenza (che ci sta, ma allora bisognerebbe riadattare i verbali precedenti)

Reply:
in che senso ho fatto esattamente allo stesso modo di altri verbali mergiati